<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui::Pref::TPlaylistSection</class>
 <widget class="QWidget" name="Gui::Pref::TPlaylistSection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="playlist_tab">
      <attribute name="title">
       <string>&amp;Playlist</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <widget class="QGroupBox" name="directory_group">
         <property name="title">
          <string>When opening a directory:</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="recursive_check">
            <property name="text">
             <string>Add files in subdirectories &amp;recursively</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="video_check">
            <property name="text">
             <string>Add &amp;video</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="audio_check">
            <property name="text">
             <string>Add a&amp;udio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="playlists_check">
            <property name="text">
             <string>Add &amp;playlists</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="images_check">
            <property name="text">
             <string>Add i&amp;mages</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="image_group">
         <property name="title">
          <string>Images</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="image_duration_label">
              <property name="text">
               <string>&amp;Display images for</string>
              </property>
              <property name="buddy">
               <cstring>image_duration_spinbox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="image_duration_spinbox">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>999</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>seconds</string>
              </property>
              <property name="buddy">
               <cstring>image_duration_spinbox</cstring>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="directorie_playlist_group">
         <property name="title">
          <string>Directorie playlists</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>When checked the playlist for a directorie will be automatically saved to wzplayer.m3u8 inside the directorie. It will be updated when opening a directorie and automatically saved when closing the playlist. It stores the names, durations and items to ignore for this directorie. Subdirectories will be linked and get their own playlist.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>directory_playlist_check</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="directory_playlist_check">
            <property name="text">
             <string>Automatically &amp;save directorie playlists</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="blacklist_tab">
      <attribute name="title">
       <string>&amp;Blacklists</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>16</number>
       </property>
       <property name="margin">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Here you can exclude files, directories and titles matching a regular expression. The expressions are case insensitive. Enter one per line.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="name_group">
         <property name="title">
          <string>Name blacklist</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="name_blacklist_label">
            <property name="text">
             <string>&amp;Don't add files or directories matching the following expressions. They are matched against the full filename including its path.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>name_blacklist_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="name_blacklist_edit">
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="title_blacklist_group">
         <property name="title">
          <string>Title blacklist</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="title_blacklist_label">
            <property name="text">
             <string>Drop &amp;titles matching the following expressions: The file name will be used as title instead.</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="buddy">
             <cstring>title_blacklist_edit</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="title_blacklist_edit">
            <property name="tabChangesFocus">
             <bool>true</bool>
            </property>
            <property name="plainText">
             <string notr="true"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>recursive_check</tabstop>
  <tabstop>video_check</tabstop>
  <tabstop>audio_check</tabstop>
  <tabstop>playlists_check</tabstop>
  <tabstop>images_check</tabstop>
  <tabstop>image_duration_spinbox</tabstop>
  <tabstop>directory_playlist_check</tabstop>
  <tabstop>name_blacklist_edit</tabstop>
  <tabstop>title_blacklist_edit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
